common/app/views/main.scala.html

```diff
@@ -93,6 +93,10 @@
         @fragments.inlineJSNonBlocking()
 
         @fragments.analytics.base()
+
+        @if(OrielAnalyticsSwitch.isSwitchedOn) {
```

Comment:
Would it be clearer to put both conditions here?

Reply:
@kelvin-chappell As I expand out into sections or more articles then I would rather have it in a fragment. If it goes further site-wide then I will probably move the logic to here.